dapp_user/application/handlers/user_handlers.py

```diff
@@ -51,3 +51,10 @@ def delete_user(event, context):
 @exception_handler(SLACK_HOOK=SLACK_HOOK, NETWORK_ID=NETWORK_ID, logger=logger)
 def register_user_post_aws_cognito_signup(event, context):
     logger.info(f"Post aws cognito sign up event {event}")
+    user_service = UserService()
+    response = user_service.register_user(user_attribute=event["request"]["userAttributes"],
```

Comment:
What happens if register_user throws an exception or if it returns an error

Reply:
Chances are rare that it will fail. But If it fails, user data will not be persisted and he will not able to pass the verification code test even if he puts the right verification code.

Reply:
We will get a slack alert on failure.

Reply:
OK, good that slack alert will be fired. Lets also send a response of error if an exception occurs or if an error is returned

Reply:
Done